Formula/exiftool.rb

```diff
@@ -30,7 +30,7 @@ def install
     inreplace "lib/Image/ExifTool.pm", "LargeFileSupport => undef", "LargeFileSupport => 1"
 
     # replace the hard-coded path to the lib directory
-    inreplace "exiftool", "$1/lib", libexec/"lib"
+    inreplace "exiftool", "$exeDir/lib", libexec/"lib"
```

Comment:
The behavior differs when run using actual script file vs. when run using a symlink, e.g.
```console
❯ /usr/local/Cellar/exiftool/12.42/bin/exiftool -ver
12.42

❯ /usr/local/bin/exiftool -ver
Can't locate Image/ExifTool.pm in @INC (you may need to install the Image::ExifTool module) (@INC contains: /usr/local/bin/../Cellar/exiftool/12.42/bin/lib /Library/Perl/5.30/darwin-thread-multi-2level /Library/Perl/5.30 /Network/Library/Perl/5.30/darwin-thread-multi-2level /Network/Library/Perl/5.30 /Library/Perl/Updates/5.30.3 /System/Library/Perl/5.30/darwin-thread-multi-2level /System/Library/Perl/5.30 /System/Library/Perl/Extras/5.30/darwin-thread-multi-2level /System/Library/Perl/Extras/5.30) at /usr/local/bin/exiftool line 40.
BEGIN failed--compilation aborted at /usr/local/bin/exiftool line 40.
```

The original `inreplace` fixes the former. This is why CI is now failing since Homebrew's test block uses the real path.

So, you probably want to have both modifications.

Reply:
Thanks @cho-m :+1: I've updated the commit to take a slightly difference approach that seems to fix both scenarios 🙏